lib/public/components/common/table/rows.js

```diff
@@ -41,7 +43,7 @@ const cell = (rowId, column, rowData) => {
     }
 
     // Keep value.size for retro-compatibility purpose
-    return h(`td.${column.size} ${column.classes}`, {
+    return h(errorStatus ? `td.${column.size} ${column.classes}.danger` : `td.${column.size} ${column.classes}`, {
```

Comment:
As said before, I think this should not be there. But FYI, with conditionals I prefer to limit the conditional branches to the most atomic things, here I would replace
```js
errorStatus ? `td.${column.size} ${column.classes}.danger` : `td.${column.size} ${column.classes}`
```
by
```js
`td.${column.size} ${column.classes}${errorStatus ? '.danger' : ''`
```

To avoid dupplication of `td.${column.size} ${column.classes}`